drivers/hmis/app/models/hmis/project_auto_exit_config.rb

```diff
@@ -5,6 +5,37 @@
 ###
 
 class Hmis::ProjectAutoExitConfig < Hmis::ProjectConfig
-  # This class is not yet used, but will be replacing AutoExitConfig
-  # TODO: Validate that config_options is not null and length_of_absence_days is defined on that json blob
+  validate :length_of_absence_days_ge_30
+
+  def length_of_absence_days=(value)
+    self.config_options = { 'length_of_absence_days': value }.to_json
+  end
+
+  def length_of_absence_days
+    JSON.parse(config_options)['length_of_absence_days']
+  end
+
+  def length_of_absence_days_ge_30
+    unless config_options
+      errors.add(:base, 'config_options must be present')
+      return
+    end
+
+    begin
+      json_blob = JSON.parse(config_options)
+    rescue JSON::ParserError
+      errors.add(:base, 'config_options must be JSON')
+      return
+    end
+
+    length_of_absence_days = json_blob['length_of_absence_days']
+    unless length_of_absence_days.is_a? Integer
+      errors.add(:base, 'config_options must contain an integer length_of_absence_days')
+      return
+    end
+
+    return unless length_of_absence_days < 30
+
+    errors.add(:base, 'length_of_absence_days must greater than or equal to 30')
```

Comment:
Rails errors should be human-readable:
```suggestion
    errors.add(:base, 'length of absence days must greater than or equal to 30')
```

We do display them directly to the user:
<img width="659" alt="Screenshot 2024-02-26 at 5 19 21 PM" src="https://github.com/greenriver/hmis-warehouse/assets/5333982/460d2ee1-5d98-476b-a081-f50fd97c4ae1">



It's not perfectly executed in this case, because the input field isn't tied to the error message. But I think that will come later with re-designing the AutoExit UI into a more general project configuration tool.